pinot-query-runtime/src/test/resources/queries/SelectHaving.json

```diff
@@ -0,0 +1,72 @@
+{
+  "select_expression_test": {
+    "tables": {
+      "test_having": {
+        "schema":[
+          {"name": "a", "type": "INT"},
+          {"name": "b", "type": "INT"},
+          {"name": "c", "type": "STRING"},
+          {"name": "d", "type": "STRING"}
+        ],
+        "inputs": [
+          [0, 1, "XXXX", "A"],
+          [1, 2, "AAAA", "b"],
+          [2, 2, "AAAA", "c"],
+          [3, 3, "BBBB", "D"],
+          [4, 3, "BBBB", "e"],
+          [5, 3, "bbbb", "F"],
+          [6, 4, "cccc", "g"],
+          [7, 4, "cccc", "h"],
+          [8, 4, "CCCC", "I"],
+          [9, 4, "CCCC", "j"]
+        ]
+      }
+    },
+    "queries": [
+      {
+        "sql":"SELECT b, c FROM test_having GROUP BY b, c HAVING count(*) = 1 ORDER BY b, c;",
+        "ignored": true
+      },
+      {
+        "sql":"SELECT b, c FROM test_having GROUP BY b, c HAVING b = 3 ORDER BY b, c;",
+        "ignored": true
+      },
+      {
+        "sql":"SELECT lower(c), count(c) FROM test_having GROUP BY lower(c) HAVING count(*) > 2 OR min(a) = max(a) ORDER BY lower(c);",
+        "ignored": true
+      },
+      {
+        "sql":"SELECT c, max(a) FROM test_having GROUP BY c HAVING count(*) > 2 OR min(a) = max(a) ORDER BY c;",
+        "ignored": true
+      },
+      {
+        "sql":"SELECT min(a), max(a) FROM test_having HAVING min(a) = max(a);",
```

Comment:
Is this for invalid / unsupported queries ?

Afaik, use of HAVING without GROUP BY is not a valid SQL syntax. I think Postgres, MySQL all support HAVING for GROUP BY aggregation queries.

Reply:
+1 if it's ignored it'd be good to add a comment as to why - also i thought we do support HAVING in some situations? (I tested it in some other test, I thought)

Reply:
Yeah. Somehow I thought we don't support join and I made a mistake with the table name.
Updated the queries and comments on where we failed.